website/docs/reference/function-configs.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Function configurations
+description: "Read this guide to learn about using function configurations in dbt."
+meta:
+  resource_type: Functions
+---
+
+import ConfigResource from '/snippets/_config-description-resource.md';
+import ConfigGeneral from '/snippets/_config-description-general.md';
+
+
+## Available configurations
+### Function-specific configurations
+
+<ConfigResource meta={frontMatter.meta} />
+
+<Tabs
+  groupId="config-languages"
+  defaultValue="project-yaml"
+  values={[
+    { label: 'Project file', value: 'project-yaml', },
+    { label: 'Property file', value: 'property-yaml', },
+  ]
+}>
+<TabItem value="project-yaml">
+
+<File name='dbt_project.yml'>
+
+```yml
+functions:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    # Function-specific configs are defined in the property file
+    # See functions/schema.yml examples below
+
+```
+
+</File>
+
+</TabItem>
+
+
+<TabItem value="property-yaml">
+
+<File name='functions/schema.yml'>
+
+```yaml
+version: 2
+
+functions:
+  - name: [<function-name>]
+    config:
+      # Standard configs that apply to functions
+      [database](/reference/resource-configs/database): <string>
+      [schema](/reference/resource-properties/schema): <string>
+      [alias](/reference/resource-configs/alias): <string>
+      [tags](/reference/resource-configs/tags): <string> | [<string>]
+      [meta](/reference/resource-configs/meta): {<dictionary>}
+
+```
+
+</File>
+
+</TabItem>
+
+</Tabs>
+
+### General configurations
+
+<ConfigGeneral />
+
+<Tabs
+  groupId="config-languages"
+  defaultValue="project-yaml"
+  values={[
+    { label: 'Project file', value: 'project-yaml', },
+    { label: 'Property file', value: 'property-yaml', },
+  ]
+}>
+
+<TabItem value="project-yaml">
+
+<File name='dbt_project.yml'>
+
+```yaml
+functions:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    [+](/reference/resource-configs/plus-prefix)[enabled](/reference/resource-configs/enabled): true | false
```

Comment:
@graciegoheen and @QMalcolm are these correct?

Reply:
This list of configs looks good to me 👍🏻

Reply:
@QMalcolm what about `identifier`?